07_deploy_masters.sh

```diff
@@ -63,6 +63,9 @@ while [[ ! $(timeout -k 9 5 $SSH "core@api.${CLUSTER_NAME}.${BASE_DOMAIN}" hostn
   sleep 10
 done
 
+#Adding some delay so that all 3 masters come up
+sleep 10
```

Comment:
Maybe it was just coincidence that this seemed to help?  Since the loop below is an infinite loop and has its own sleep, I don't imagine this changes things?  Unless calling "oc get nodes" 10 seconds later for the first time helps?  That would be pretty weird though ...

In any case, I think it would make sense in a separate PR

Reply:
I've seen the same error, the issue is the set -e means the first failure causes the script to exit so we never retry.

So we either need to make the VIP healtcheck better so it doesn't fail-over the VIP until the API is actually ready, tolerate failure in the oc calls, or add some sleep/retry here.

Agree it would probably be better in a separate patch though.

Reply:
Actually this will probably be fixed via https://github.com/openshift-metalkube/dev-scripts/pull/130 if we can figure out how to land that and not break CI...

Reply:
Possible that it is coincidence. As soon as make failed at this point I was able to actually connect to the API at the command line. So, added this is a temporary workaround so that CI could actually get to my code in 08_deploy_bmo.sh. (And it did, in the run with this change in place.) I had patched in #130 during my tests and did get better results.